.travis.yml

```diff
@@ -3,11 +3,28 @@ language: node_js
 
 sudo: false
 
+node_js:
+  - "0.10"
+  - "0.12"
+  - "iojs"
+
 cache:
   directories:
     - node_modules
 
+env:
+  - NPM_SCRIPT=node-test
+  - NPM_SCRIPT=client-test
+
+matrix:
+  allow_failures:
+    - node_js: "0.10"
```

Comment:
We use Node.js `0.10.x` internally at Yahoo, so I'd like to get this passing at some point in the future.